src/ducks/GainersAndLosers/GainersLosersGraph.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { LineChart, Line } from 'recharts'
+import { graphql } from 'react-apollo'
+import { SOCIAL_GAINERS_LOSERS_STATUS_QUERY } from './gainersLosersQuery'
+import { getTimeRangeByDuration } from '../../utils/utils'
+
+const GainersLosersGraph = ({ socialGainersLosersStatus }) => (
+  <LineChart
+    width={300}
+    height={50}
+    data={socialGainersLosersStatus}
+    margin={{ top: 5, right: 20, left: 20, bottom: 5 }}
+  >
+    <Line
+      type='monotone'
+      dataKey='change'
+      stroke='var(--grey-dark)'
```

Comment:
There is no `grey-dark` in our color palette. Please, check the color by hex if the name corresponding does not exist